helpers/uuid/uuid.go

```diff
@@ -0,0 +1,23 @@
+package uuid
+
+import (
+	"crypto/rand"
+	"encoding/hex"
+	"hash/fnv"
+)
+
+func UUID() (string, error) {
```

Comment:
This is not a UUID as defined in several standards. https://en.wikipedia.org/wiki/Universally_unique_identifier.

Can we call this something else? I expect UUIDs to have a certain length and format.

Reply:
Sure. ... Package rename to `randstr` and `UUID` becomes `Hex16`. Noted.

Reply:
Nice, that means it doesn't "stutter"   (https://golang.org/doc/effective_go#package-names) anymore. Instead of `uuid.UUID()`, we have `randstr.Hex16()` 👍

Reply:
Done with commit 28f8364ac94c1a734c0d50c71034a6e3be51b337